node/network/bridge/src/lib.rs

```diff
@@ -128,7 +128,7 @@ impl<N, AD> NetworkBridge<N, AD> {
 impl<Net, AD, Context> NetworkBridge<Net, AD>
 where
 	Net: Network + Sync,
-	AD: validator_discovery::AuthorityDiscovery + Clone,
+	AD: validator_discovery::AuthorityDiscovery + Clone + Sync,
```

Comment:
not sure why `Sync` is now required, but it's probably fine

Reply:
I could not convince the compiler otherwise, our current impl is `Sync` 🤷
I can try again with a fresh pair of 👀

Reply:
After cleaning up everything the sync bounds were removable, probably my fault.